FWCore/Integration/test/test_TestInterProcessProd_cfg.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
+
+
+class ThingExternalProcessProducer(cms.EDProducer):
+    def __init__(self, prod):
+        self.__dict__['_prod'] = prod
+        super(cms.EDProducer,self).__init__('TestInterProcessProd')
+    def __setattr__(self, name, value):
+        setattr(self._prod, name, value)
+    def __getattr__(self, name):
+        if name =='_prod':
+            return self.__dict__['_prod']
+        return getattr(self._prod, name)
+    def clone(self, **params):
+        returnValue = ThingExternalProcessProducerProducer.__new__(type(self))
+        returnValue.__init__(self._prod)
+        return returnValue
+    def insertInto(self, parameterSet, myname):
+        newpset = parameterSet.newPSet()
+        newpset.addString(True, "@module_label", self.moduleLabel_(myname))
+        newpset.addString(True, "@module_type", self.type_())
+        newpset.addString(True, "@module_edm_type", cms.EDProducer.__name__)
+        newpset.addString(True, "@external_type", self._prod.type_())
+        newpset.addString(False,"python_config", self._prod.dumpPython())
```

Comment:
I wonder if `python_config` should also start with `@`? In `SwitchProducer` all "internal" parameters start with `@`.

Reply:
The parameter is untracked so it doesn't matter either way.

Reply:
I'm more thinking about a name collision with the module parameters (ok, this is still a fully controlled test), and for that we seem to use prefix `@` for framework-internal parameters.